[2023-10-01 06:14:41,834] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_fourth_dag_v4.first_task 2021-10-27T00:00:00+00:00 [queued]>
[2023-10-01 06:14:42,232] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_fourth_dag_v4.first_task 2021-10-27T00:00:00+00:00 [queued]>
[2023-10-01 06:14:42,236] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-10-01 06:14:42,240] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2023-10-01 06:14:42,246] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-10-01 06:14:42,847] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): first_task> on 2021-10-27T00:00:00+00:00
[2023-10-01 06:14:44,084] {standard_task_runner.py:52} INFO - Started process 5222 to run task
[2023-10-01 06:14:44,359] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_fourth_dag_v4', 'first_task', '2021-10-27T00:00:00+00:00', '--job-id', '693', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_fourth_dag.py', '--cfg-path', '/tmp/tmpym3hhh58', '--error-file', '/tmp/tmptmp6tv9_']
[2023-10-01 06:14:44,508] {standard_task_runner.py:77} INFO - Job 693: Subtask first_task
[2023-10-01 06:14:45,376] {logging_mixin.py:104} INFO - Running <TaskInstance: our_fourth_dag_v4.first_task 2021-10-27T00:00:00+00:00 [running]> on host d4ed655215d1
[2023-10-01 06:14:47,226] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=svastikkka
AIRFLOW_CTX_DAG_ID=our_fourth_dag_v4
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-27T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-27T00:00:00+00:00
[2023-10-01 06:14:47,241] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-10-01 06:14:47,274] {bash.py:158} INFO - Running command: echo hello world, this is the first task!
[2023-10-01 06:14:47,330] {bash.py:169} INFO - Output:
[2023-10-01 06:14:47,355] {bash.py:173} INFO - hello world, this is the first task!
[2023-10-01 06:14:47,369] {bash.py:177} INFO - Command exited with return code 0
[2023-10-01 06:14:48,058] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_fourth_dag_v4, task_id=first_task, execution_date=20211027T000000, start_date=20231001T061441, end_date=20231001T061448
[2023-10-01 06:14:48,568] {taskinstance.py:1220} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2023-10-01 06:14:49,013] {local_task_job.py:146} INFO - Task exited with return code 0
